src/ai/backend/storage/config/loaders.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import os
+import sys
+from pathlib import Path
+from pprint import pformat
+
+from ai.backend.common.config import (
+    override_key,
+    override_with_env,
+    read_from_file,
+)
+from ai.backend.common.etcd import AsyncEtcd, ConfigScopes
+
+from .unified import StorageProxyUnifiedConfig
+
+
+def load_local_config(config_path: Path | None, debug: bool = False) -> StorageProxyUnifiedConfig:
+    """Load and validate the storage-proxy local configuration."""
+    # Determine where to read configuration
+    raw_cfg, cfg_src_path = read_from_file(config_path, "storage-proxy")
+    os.chdir(cfg_src_path.parent)
+
+    # Apply environment overrides
+    override_with_env(raw_cfg, ("etcd", "namespace"), "BACKEND_NAMESPACE")
+    override_with_env(raw_cfg, ("etcd", "addr"), "BACKEND_ETCD_ADDR")
+    override_with_env(raw_cfg, ("etcd", "user"), "BACKEND_ETCD_USER")
+    override_with_env(raw_cfg, ("etcd", "password"), "BACKEND_ETCD_PASSWORD")
+
+    if debug:
+        override_key(raw_cfg, ("debug", "enabled"), True)
+
+    try:
+        local_config = StorageProxyUnifiedConfig.model_validate(raw_cfg)
+        return local_config
```

Comment:
Does `UnifiedConfig` meant LocalConfig + SharedConfig?
This seems not Unified

Reply:
This time, I only applied pydantic, but it needs to be applied further later. (The task of merging for the purpose you mentioned needs to be added.)
For now, I worked on it because converting to pydantic is necessary to generate toml based on the model schema.